### PR TITLE
Feature: Create code of conduct

### PR DESCRIPTION
**Pull request by annakijas1**
Created code of conduct page under "Community." Need to add to page to the menu under "Community"

### CODE REVIEW

File: community/code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Music Encoding Initiative Community Code of Conduct
+
+## **1. Purpose**
```
Comment (bwbohl) [inline community/code-of-conduct.md:3]: here and at all other headings the asterisks for marking text bold are not needed, as the styles of the page will automatically render them bold.

```suggestion
## 1. Purpose
```

File: community/code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+* Exercise consideration and respect in your speech and actions.
+
+* Attempt collaboration before conflict.
+
+* Refrain from demeaning, discriminatory, or harassing behavior and speech.
+
+* Be mindful of your surroundings and of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
+
+* Remember that community event venues may be shared with members of the public; please be respectful to all patrons of these locations.
```
Comment (bwbohl) [inline community/code-of-conduct.md:33]: The empty lines between the list bullets are not needed.  The resulting HTML list is valid. Nevertheless, the list entries will each be wrapped in an HTML paragraph element which will take up more space. If that is your intention, it is fine with me, but usually, we have more compact lists on the website (accomplished by leaving away the additional line break).

```suggestion
* Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
* Exercise consideration and respect in your speech and actions.
* Attempt collaboration before conflict.
* Refrain from demeaning, discriminatory, or harassing behavior and speech.
* Be mindful of your surroundings and of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
* Remember that community event venues may be shared with members of the public; please be respectful to all patrons of these locations.
```

You can compare the result in the following example:

* this is a list with

* paragraphs

* caused by the additional line-breaks

vs.

* a more
* compact list
* without additional empty lines

**Review by bwbohl — CHANGES_REQUESTED** [review]
the two comments and suggestions I made also apply to all other headings and lists.

**Comment by bwbohl** [comment]
Adding the page to the submenu can be done by adding it in the [_config.yml](https://github.com/music-encoding/music-encoding.github.io/blob/33d51ee125a72ca33d334d4f2c1a00ac8b205b93/_config.yml) of your branch.

The submenu of the conference menu entry is located at:
https://github.com/music-encoding/music-encoding.github.io/blob/33d51ee125a72ca33d334d4f2c1a00ac8b205b93/_config.yml#L23-L44

File: community/code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+This Code of Conduct and its related procedures also applies to unacceptable behavior occurring outside the scope of community activities when such behavior has the potential to adversely affect the safety and well-being of community members.
+
+## **9. License and attribution**
+
+The Music Encoding Initiative Community Code of Conduct is based on the [Citizen Code of Conduct](https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md) distributed by[ ](http://stumptownsyndicate.org)[Stumptown Syndicate](http://stumptownsyndicate.org) under a[ ](http://creativecommons.org/licenses/by-sa/3.0/)[Creative Commons Attribution-ShareAlike license](http://creativecommons.org/licenses/by-sa/3.0/). Portions of text derived from the[ ](https://www.djangoproject.com/conduct/)[Django Code of Conduct](https://www.djangoproject.com/conduct/) and the[ ](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy)[Geek Feminism Anti-Harassment Policy](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy).
```
Comment (musicEnfanthen) [inline community/code-of-conduct.md:97]:
```suggestion
The Music Encoding Initiative Community Code of Conduct is based on the [Citizen Code of Conduct](https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md) distributed by [Stumptown Syndicate](http://stumptownsyndicate.org) under a [Creative Commons Attribution-ShareAlike license](http://creativecommons.org/licenses/by-sa/3.0/). Portions of text derived from the [Django Code of Conduct](https://www.djangoproject.com/conduct/) and the [Geek Feminism Anti-Harassment Policy](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy).
```

**Review by musicEnfanthen — CHANGES_REQUESTED** [review]

**Review by musicEnfanthen — APPROVED** [review]

**Comment by musicEnfanthen** [comment]
@annakijas1 @bwbohl I added the code-of-conduct page to the config.yml and also the Jekyll frontend at beginning of the code's page directly to this branch, so the page should be displayed when published (checked locally). Please have a look.

**Comment by musicEnfanthen** [comment]
@bwbohl Are you fine with the changes?

**Pull request merged**